lib/Sat/CryptoMinisat5.cpp

```diff
@@ -169,7 +169,7 @@ uint32_t CryptoMiniSat5::getFixedCountWithAssumptions(const stp::SATSolver::vec_
 
   //std::cerr << assumps.size() << " assumptions" << std::endl;
 
-  unsigned assigned = 0;
+  int assigned = 0;
```

Comment:
This change isn't right, as `assigned` can never be negative.

Is it `assigned >= assumps.size()` that's warning?

Reply:
Yes, because `assumps.size()` can't be negative. I can cast `assigned`, but it compared with integer in any way. So using `unsigned` doesn't make sense to me.

Reply:
Cast the result of `assumps.size()` to `unsigned int`.